README.md

```diff
@@ -15,6 +15,31 @@ Current status:
 
 See the [docs](https://github.com/MastodonC/witan.models.demography/blob/master/doc/intro.md) for more info about the methodology.
 
+## Development tools
+
+The `:dev` leiningen profile includes a dependency to the library `witan.workspace-executor`.
+This library contains tool that are useful for developer contributing to `witan.models.demography`.
+
+* visualise your model workflow:
+
+You need to have `Graphviz` installed:
+
+-> Ubuntu: `> sudo apt-get install graphviz`
+
+-> MacOS: `> brew install graphviz`
+
+For any OS you should also be able to install it with Pip: `pip install graphviz`.
+
+```Clojure
+(witan.workspace-executor/view-workkflow my-workflow)
```

Comment:
It's not immediately clear that this: `* visualise your model workflow:` is associated with the above command, consider moving things about so it is a little clearer